src/DotLiquid/Util/ObjectExtensionMethods.cs

```diff
@@ -11,6 +11,9 @@ public static bool RespondTo(this object value, string member, bool ensureNoPara
             if (value == null)
                 throw new ArgumentNullException("value");
 
+            if (value is DropBase)
+                return (value as DropBase).ContainsKey(member);

```

Comment:
to avoid repeat type casting C# allows in current version to use the shortcut
if (value is DropBase dropBase)
 return dropBase.FooBar();